easybuild/framework/package.py

```diff
@@ -0,0 +1,65 @@
+# #
+# Copyright 2009-2014 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+# #
+
+"""
+A place for packaging functions
+
+"""
+
+import os
+from easybuild.tools.run import run_cmd
+
+def package_fpm(easyblock, modfile_path ):
+    rpmname = "HPCBIOS.20150211-%s-%s" % (easyblock.name, easyblock.version)
+    workdir = "/tmp"
+    os.chdir(workdir)
+
+    pkgtemplate = "HPCBIOS.20150211-%(name)s-%(version)s"
```

Comment:
it's a template name, each for his own ;-)
this belongs to an external definition (possibly eb configfile), be my guest